### PR TITLE
7505 txn created with error from qb

### PR DESCRIPTION
**Pull request by hooverlunch**

### CODE REVIEW

**Comment by emitche** [comment]
I came across a bug while testing this in my environment. Please see: https://redmine.sassafras.coop/issues/7505#note-12.

File: app/models/accounting/quickbooks/updater.rb [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-    class FullSyncRequiredError < StandardError; end
-    class NotConnectedError < StandardError; end
 
     # Reponsible for grabbing only the updates that have happened in quickbooks
     # since the last time this class was run. If no quickbooks data exists in the sytem
     # yet, FullFetcher will need to be run first.
     class Updater
       attr_reader :qb_connection
 
+      MIN_TIME_BETWEEN_UPDATES = 5.seconds
```
Comment (arisah-ijeoma) [inline app/models/accounting/quickbooks/updater.rb]: I thought this would be 15 seconds

Comment (hooverlunch) [reply app/models/accounting/quickbooks/updater.rb]: Yes, I reconsidered. There is a comment about this elsewhere in the file I think.

**Comment by emitche** [comment]
Even with a fresh database copy, I still am unable to load transactions. Going to the settings page and running a full sync gives me the same error. More info in Redmine. Thanks.

**Comment by emitche** [comment]
I believe the code is ready for this branch. The problem is a Rails bug that blocks this branch. See https://redmine.sassafras.coop/issues/7684.

**Comment by emitche** [comment]
I can't really QA this, so I think this is ready, given that tests are passing.

**Pull request closed without merging**